src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs

```diff
@@ -98,6 +111,7 @@ public override void ExecuteCmdlet()
                         { ItemParams.Container, Container },
                         { ItemParams.BackupManagementType, BackupManagementType },
                         { ItemParams.AzureVMName, Name },
+                        { PolicyParams.ProtectionPolicy, Policy},
```

Comment:
Do "Ctrl + K + D" on all the modified files to ensure proper formatting is maintained.